docs/en/sql-reference/functions/array-functions.md

```diff
@@ -1144,11 +1144,32 @@ SELECT arrayFold( x,acc -> acc + x*2,  [1, 2, 3, 4], toInt64(3)) AS res;
 Result:
 
 ``` text
-┌─arrayFold(lambda(tuple(x, acc), plus(acc, multiply(x, 2))), [1, 2, 3, 4], toInt64(3))─┐
-│                                                                                     3 │
-└───────────────────────────────────────────────────────────────────────────────────────┘
+┌─res─┐
+│  23 │
+└─────┘
+```
+
+**Example with the fibonacci sequence**
```

Comment:
```suggestion
**Example with the Fibonacci sequence**
```

Reply:
I am not sure if the the aspell dict has to be extended with `Fibonacci` or not.